docs/content/en/blog/articles/minio-backup/index.md

```diff
@@ -0,0 +1,268 @@
+
+Now refresh the MinIO UI and you should see some files in the k8ssandra-medusa bucket:
+
+![K8ssandra Backup Files](k8ssandra-medusa-backup.png)
+
+An index folder should appear (it is Medusa’s backup index) and then another folder that is specific to each Cassandra node in the cluster (in this case there is only one node).
+
+# Deleting the data and restoring the backup
+
+Now delete the data by truncating the table, and check that the table is empty:
+
+```
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -e "TRUNCATE medusa_test.users"
+
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -e "SELECT * FROM medusa_test.users"
+
+ email | name | state
+-------+------+-------
+
+(0 rows)
+```
+
+Now restore the backup taken previously:
+
+```
+helm install restore-test k8ssandra/restore --set name=restore-backup1,backup.name=backup1,cassandraDatacenter.name=dc1 -n k8ssandra
+```
+
+This operation will take a little longer as it requires to stop the StatefulSet pod and perform the restore as part of the init containers, before the Cassandra container can start. You can monitor progress using this command:
+
+```
+watch -d kubectl get cassandrarestore restore-backup1 -o jsonpath={.status} -n k8ssandra
+```
+
+The restore operation is fully completed once the **finishTime** value appears in the output:
+
+```
+{"finishTime":"2021-03-23T13:58:36Z","restoreKey":"83977399-44dd-4752-b4c4-407273f0339e","startTime":"2021-03-23T13:55:35Z"}
+```
+
+Check that you can read the data from the previously truncated table:
+
+```
+% kubectl exec -it k8ssandra-dc1-default-sts-0 -n k8ssandra -c cassandra -- cqlsh -u k8ssandra-superuser -p XHsZ943WBg5RPNhVAT8x -e "SELECT * FROM medusa_test.users"
+
+ email             | name          | state
+-------------------+---------------+-------
+ alice@example.com |   Alice Smith |    TX
+   bob@example.com |     Bob Jones |    VA
+ david@example.com |    David Yang |    NV
+ carol@example.com | Carol Jackson |    CA
+
+(4 rows)
+```
+
+You’ve successfully restored your lost data in just a few commands!
+
+# Many backends available
+
+MinIO, while being an obvious choice in the Kubernetes world, is not the only S3 compatible backend that K8ssandra can use. K8ssandra has supported AWS S3 and Google Cloud Storage as Medusa backends since 1.0.0. There is also a wide variety of solutions that can run on-prem (including CEPH, Cloudian, Riak S2, and Dell EMC ECS) or in cloud environments (including IBM Cloud Object Storage, and OVHcloud Object Storage). See the [K8ssandra backup/restore documentation](https://k8ssandra.io/docs/topics/restore-a-backup/) for more detailed instructions and let us know if you have questions, we love to help!
```

Comment:
How should they let us know?

Reply:
yeah, how would they @jeffreyscarpenter? 🙂 
We need an ASF Slack channel... 😇

Reply:
We'll be launching forums/chat soon. We can update this post then. If you want to direct to the email list in the meantime maybe that's the best?

Reply:
I see a few options to choose from today

- User list (k8ssandra-users@googlegroups.com)
- Twitter (@k8ssandra)
- GH Issues

My ask is that we include at least 1 and call this comment resolved.

Reply:
let's go with user list. Can you push that change along with the CQL change @adejanovski ?